index.html

```diff
@@ -1,33 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
-    <link rel="stylesheet" href="style.css">
-    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+    />
     <title>Portoflio</title>
   </head>
   <body>
+    <!-- HEADER -->
     <header>
       <nav class="header_name">
-        <a href="/" class="top"><h2 id="top" >Olivier</h2></a>   
-        <a href="main.html" class="container_menu" onclick="myFunction(this)">
+        <a href="#" class="top"><h2 id="top">Olivier</h2></a>
+        <ul class="li-desk">
+          <li class="nav-item"><a class="nav-link" href="#">Portoflio</a></li>
+          <li class="nav-item"><a class="nav-link" href="#recent-work">About</a></li>
+          <li class="nav-item"><a class="nav-link" href="#contact">Contact</a></li>
```

Comment:
In the nav menu, the `portfolio` link takes to the about section and the `about` link to the portfolio section. Kindly fix this issue and submit it for a re-review.